flows/vision-zero/cris_import/cris_import.py

```diff
@@ -244,13 +243,14 @@ def pgloader_csvs_into_database(directory):
                 command_file = pgloader_command_files_tmpdir + "/" + table + ".load"
                 print(f'Command file: {command_file}')
 
-                CONNECTION_STRING = f'postgresql://{DB_USER}:{DB_PASS}@{DB_HOST}:5432/{DB_NAME}'
+                # See https://github.com/dimitri/pgloader/issues/768#issuecomment-693390290
+                CONNECTION_STRING = f'postgresql://{DB_USER}:{DB_PASS}@{DB_HOST}:5432/{DB_NAME}?sslmode=allow'
 
                 with open(command_file, 'w') as file:
                     file.write(f"""
 LOAD CSV
     FROM '{directory}/{filename}' ({headers_line})
-    INTO  {CONNECTION_STRING}?import.{table} ({headers_line})
+    INTO  {CONNECTION_STRING}&import.{table} ({headers_line})
```

Comment:
For future reviewers - this PR ended up with a lot of extransious commits from our current branching configuration in `atd-prefect`. The above lines are the fix which he came up with. 

The two commits of actual interest in this PR are:
* https://github.com/cityofaustin/atd-prefect/pull/60/commits/99334e6ac79dd1dbb7c2b45a3238844f5452acc8
* https://github.com/cityofaustin/atd-prefect/pull/60/commits/99334e6ac79dd1dbb7c2b45a3238844f5452acc8